src/Models/Link.php

```diff
@@ -46,6 +47,8 @@ class Link extends DataObject
         ],
     ];
 
+    private static $default_sort = '"Sort"';
```

Comment:
```suggestion
    private static $default_sort = 'Sort';
```
Shouldn't need `"` here iirc. It's not passed into a raw SQL method.

Reply:
Updated